lib/support/load_helper.rb

```diff
@@ -0,0 +1,35 @@
+# Helper to facilitate errorlessly loading the classes aliased by ActiveEmoji
+
+# List of files for all ActiveEmoji aliased classes
+FILELIST = Dir[File.dirname(__FILE__) + '/active_emoji/core_ext/*.rb']
+
+def load_all_loadable_classes
+  # Load aliases for all classes that exist in memory at runtime
+  if rails_loaded?
+    load_all_classes
+  else
+    load_all_classes_except 'date.rb'
+  end
+end
+
+
+
+# Constituent methods of load_all_loadable_classes
+
+def rails_loaded?
+  # TODO: Extend to accept top-level constants for any library as argument
+  defined?(Rails) ? true : false
```

Comment:
@vendethiel That's legit. I mulled it over more and came round to your way of thinking.

Anyway, not sure the interest is there on this one, so I'm inclined to close it for now